src/main/java/it/gov/pagopa/payhub/activities/service/treasury/TreasuryOpi14MapperService.java

```diff
@@ -0,0 +1,73 @@
+package it.gov.pagopa.payhub.activities.service.treasury;
+
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.OrganizationDTO;
+import it.gov.pagopa.payhub.activities.dto.treasury.TreasuryDTO;
+import it.gov.pagopa.payhub.activities.enums.TreasuryOperationEnum;
+import it.gov.pagopa.payhub.activities.util.TreasuryUtils;
+import it.gov.pagopa.payhub.activities.xsd.treasury.opi14.FlussoGiornaleDiCassa;
+import it.gov.pagopa.payhub.activities.xsd.treasury.opi14.InformazioniContoEvidenza;
+
+import java.util.*;
+
+public class TreasuryOpi14MapperService implements TreasuryMapperService<FlussoGiornaleDiCassa, Map<TreasuryOperationEnum, List<TreasuryDTO>>> {
```

Comment:
Instead of handling a List to transform into a map, let's handle a single item at time.
The mapped type should be the same for all mappers, the reduce operatori ti build the final map could be implementer once on the strategy class